### PR TITLE
Supplier registers as a tenant (with a faked onelogin call) - Frontend

### PR DESCRIPTION
**Pull request by dharmverma**

### CODE REVIEW

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/OneLoginCallback.cshtml.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+public class OneLoginCallback(
+    IOneLoginClient oneLoginClient,
+    ITenantClient tenantClient,
+    ISession session) : PageModel
+{
+    public async Task<IActionResult> OnGet()
+    {
+        var userInfo = await oneLoginClient.GetUserInfo()
+                    ?? throw new Exception("Unable to retrive user info"); // show error page?
```
Comment (jakzal) [inline Frontend/CO.CDP.OrganisationApp/Pages/Registration/OneLoginCallback.cshtml.cs:17]: We should probably show an error and offer to retry. Can you add it to the backlog please?

File: Frontend/CO.CDP.OrganisationApp/appsettings.Development.json [hunk 2]
```diff
@@ -5,5 +5,6 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
-  }
+  },
+  "TenantService": "http://host.docker.internal:8080"
```
Comment (jakzal) [inline Frontend/CO.CDP.OrganisationApp/appsettings.Development.json:9]: This could just be `"http://localhost:8080"`

**Review by jakzal — APPROVED** [review]
A very tidy pull request! ❤️ 

Only minor comments. Approved preemptively.

**Review by dpatel017 — APPROVED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/OneLoginCallback.cshtml.cs (hunk 1 above)
Comment (dharmverma) [reply Frontend/CO.CDP.OrganisationApp/Pages/Registration/OneLoginCallback.cshtml.cs:17]: Created the the backlog task  - 
https://noticingsystem.atlassian.net/browse/DP-105

**Pull request merged**